lintreview/tools/flake8.py

```diff
@@ -60,7 +60,7 @@ def process_files(self, files):
         process_quickfix(self.problems, output, docker.strip_base)
 
     def make_command(self, files):
-        command = ['flake8', '--isolated']
```

Comment:
There were a number of problems without `--isolated` mode as flake8 would pick up options from `setup.cfg`, and that file would sometimes modify the output format making it impossible to be reliably parsed.

This flag *could* be removed if the `config` flag is present, but its still kind of risky as any config file with a formatter in it will break review results.

Reply:
Don't we need `--format=default` to be in the command options now?